CHANGES.rst

```diff
@@ -399,6 +399,10 @@ astropy.utils
 
 - New ``is_url`` function so downstream packages do not have to secretly use
   the hidden ``_is_url`` anymore. [#10684]
+- Added ``ssl_context`` and ``allow_insecure`` options to ``download_file``,
```

Comment:
Nitpick. The blank line must have gotten lost in rebase.
```suggestion

- Added ``ssl_context`` and ``allow_insecure`` options to ``download_file``,
```

Reply:
I guess so...